### ISSUE TITLE
How to get a list of presets/programs in soundfont

### ISSUE THREAD

**Issue by Zipdox** [jet2jet/js-synthesizer#14]
I don't think you've implemented the functions to do so. At least, your code doesn't contain any calls to `_fluid_sfont_get_preset` or `_fluid_synth_get_program`.

**Comment by jet2jet** [comment]
> I don't think you've implemented the functions to do so

Yes, currently not implemented.

I think it's useful if SoundFont APIs are included in this library, so I'll implement in next release.

**Comment by jet2jet** [comment]
I added `Synthesizer.prototype.getSFontObject` in commit 5a8c336d.
For this commit, you can retrieve Soundfont presets as follows:

```ts
    const sfontId = await synth.loadSFont(binSFont);
    const sfont = synth.getSFontObject(sfontId);
    const allPresets = [...sfont.getPresetIterable()]; // each item of allPresets will be `Preset` instance
    // for retrieving preset names:
    const allPresetNames = allPresets.map((preset) => preset.getName());
```

**Comment by Zipdox** [comment]
getSFontObject doesn't work with AudioWorkletNodeSynthesizer

**Comment by jet2jet** [comment]
Sorry I forgot AudioWorkletNodeSynthesizer...
I commited 11ee6b4b for AudioWorkletNodeSynthesizer.

**Comment by jet2jet** [comment]
I released as v1.8.0.

**Comment by jet2jet** [comment]
Closing this issue. Feel free to reopen if some problems still persist.
